sdk/keyvault/keyvault-keys/review/keyvault-keys.review.api.md

```diff
@@ -0,0 +1,295 @@
+## API Report File for "@azure/keyvault-keys"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { HttpClient } from '@azure/core-http';
+import { HttpPipelineLogger } from '@azure/core-http';
+import * as msRest from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { ServiceClientCredentials } from '@azure/core-http';
+import { ServiceClientOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface CreateEcKeyOptions extends CreateKeyOptions {
+    curve?: JsonWebKeyCurveName;
+    hsm?: boolean;
+}
+
+// @public
+export interface CreateKeyOptions {
+    enabled?: boolean;
+    expires?: Date;
+    keyOps?: JsonWebKeyOperation[];
+    // (undocumented)
+    keySize?: number;
+    notBefore?: Date;
+    requestOptions?: msRest.RequestOptionsBase;
+    tags?: {
+        [propertyName: string]: string;
+    };
+}
+
+// @public
+export interface CreateRsaKeyOptions extends CreateKeyOptions {
+    hsm?: boolean;
+    keySize?: number;
+}
+
+// @public
+export class CryptographyClient {
+    constructor(url: string, key: string | JsonWebKey, // keyUrl or JsonWebKey
+    credential: TokenCredential, pipelineOrOptions?: ServiceClientOptions | NewPipelineOptions);
+    protected readonly credential: ServiceClientCredentials | TokenCredential;
+    decrypt(algorithm: JsonWebKeyEncryptionAlgorithm, ciphertext: Uint8Array, options?: DecryptOptions): Promise<DecryptResult>;
+    encrypt(algorithm: JsonWebKeyEncryptionAlgorithm, plaintext: Uint8Array, options?: EncryptOptions): Promise<EncryptResult>;
+    static getDefaultPipeline(credential: ServiceClientCredentials | TokenCredential, pipelineOptions?: NewPipelineOptions): ServiceClientOptions;
+    getKey(options?: GetKeyOptions): Promise<JsonWebKey>;
+    key: string | JsonWebKey;
+    readonly pipeline: ServiceClientOptions;
+    // Warning: (ae-forgotten-export) The symbol "KeySignatureAlgorithm" needs to be exported by the entry point index.d.ts
+    sign(algorithm: KeySignatureAlgorithm, digest: Uint8Array, options?: RequestOptions): Promise<SignResult>;
+    signData(algorithm: KeySignatureAlgorithm, data: Uint8Array, options?: RequestOptions): Promise<SignResult>;
```

Comment:
In this case the data specified might be of unbounded size so I think Brian's comment of accepting buffer types is definitely valid in this case.

Reply:
We should do the same as what we do above for encrypt / decrypt.